terraform/services/snyk-ecr-integration/main.tf

```diff
@@ -0,0 +1,104 @@
+locals {
+  repos = {
+    ab2d = [
+      "repo:CMSgov/ab2d-bcda-dpc-platform:*",
+      "repo:CMSgov/ab2d-events:*",
+      "repo:CMSgov/ab2d-lambdas:*",
+      "repo:CMSgov/ab2d:*",
+    ]
+    bcda = [
+      "repo:CMSgov/ab2d-bcda-dpc-platform:*",
+      "repo:CMSgov/bcda-app:*",
+    ]
+    dpc = [
+      "repo:CMSgov/ab2d-bcda-dpc-platform:*",
+      "repo:CMSgov/dpc-app:*",
+    ]
+  }
+}
+
+data "aws_iam_policy" "developer_boundary_policy" {
+  name = "developer-boundary-policy"
+}
+
+data "aws_ssm_parameter" "snyk_integration_role_arn" {
+  name = "/snyk-integration/role-arn"
+}
+
+data "aws_ssm_parameter" "external_id" {
+  name = "/snyk-integration/external_id"
+}
+
+data "aws_ssm_parameter" "ecr_integration_user" {
+  name = "/snyk-integration/ecr_integration_user"
+}
```

Comment:
These SSM parameters should be created as type "Secure Text" in the AWS console.